docs/modules/ROOT/pages/index.adoc

```diff
@@ -7,3 +7,59 @@ This gem introduces two core classes of RuboCop:
 
 See xref:node_pattern.adoc["Node Pattern"] to get yourself familiar with ``NodePattern``'s
 capabilities.
+
+This gem is essentially an extension of the https://github.com/whitequark/parser/[`parser`] gem, which adds a simpler and more powerful object-oriented API to make it easier to work with nodes.
+
+Examples using `parser` and `rubocop-ast`:
```

Comment:
These are nice examples. I would be good to state somewhere that the `parser` code still works in `rubocop-ast`, and that `RuboCop::AST::Node.is?_a(Parser::AST::Node)`.